### PR TITLE
[Keyboard] Fix Redox Media compilation errors

### PR DESCRIPTION
**Pull request by drashna**
## Description

Fix errors not caught.

## Types of Changes

- [ ] Bugfix
- [ ] Keyboard (addition or update)

## Issues Fixed or Closed by This PR

* #13084

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
@shiftux
I didn't catch it, but it looks like the LAYOUT do not match.

Right now, the LAYOUT macro needs to be reduced to fit the board level macro (eg, so the default keymap compiles). 


Long term, it may be best to move it to keyboards/redox_media instead.

**Comment by shiftux** [comment]
hello
that seems strange to me. evt compiled on my end when i built this. what is more, this change would be useless... it would not respect the difference in physical layout that the media adaptations have... there are more buttons than on the usual redox.

**Comment by drashna** [comment]
> that seems strange to me. evt compiled on my end when i built this. what is more, this change would be useless... it would not respect the difference in physical layout that the media adaptations have... there are more buttons than on the usual redox.

That's likely because your fork wasn't fully up to date with the repo.  That can happen.

But you can see it erroring out, because there are not enough keys in the default keymap: 
https://qmk.tzarc.io/

File: keyboards/redox_media/rules.mk [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/redox_media/rules.mk:23]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/redox_media/readme.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Obviously start by [installing QMK](https://docs.qmk.fm/#/getting_started_build_
 
 - Before flashing the firmware you should make sure that it compiles. Thus assuring that your QMK setup works fine and the keyboard and keymap files are correct:
 ```
-qmk compile -kb redox/media -km media_ch
+qmk compile -kb redox_media -km media_ch
```
Comment (fauxpark) [inline keyboards/redox_media/readme.md:29]:
```suggestion
qmk compile -kb redox_media -km default
```
etc

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/redox_media/rules.mk [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/redox_media/rules.mk:21]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**